README + CONTRIBUTING

roznerx opened the pull request:
Added the update for the README.md file, plus a CONTRIBUTING.md empty file.

Will start adding more changes soon: let's use this PR as a way to make reviews and suggestions!

roznerx committed:
Added the update for the README.md file, plus a CONTRIBUTING.md empty file

lautapercuspain commented:
Great @roznerx! 
Thanks for putting together this PR. 🎉

lautapercuspain commented on README.md:
```diff
@@ -1,13 +1,101 @@
+- [Copyright and license](#copyright-and-license)
+
+
+## Quick start
```
We could move the quick start section to CONTRIBUTING if that makes sense.

lautapercuspain commented on README.md:
```diff
@@ -1,13 +1,101 @@
+- We are using the [Next.js App directory](https://nextjs.org/docs/app) to provide a solid skeleton for our project.
+- The aforementioned [RSC](https://nextjs.org/docs/app/building-your-application/rendering/server-components) is a key part of our website.
+- [OpenAI API](https://openai.com/blog/openai-api) is a must, being the base of Codegenius since day one!
+- Every one of our routes includes a page.tsx and client.tsx file
```
```suggested
Every one of our routes includes a page.tsx and client.tsx file. `Page.tsx` is a RSC by default, usually Client.tsx is a client component and it should use the `use client` directive at the top of the file.

lautapercuspain commented on README.md:
```diff
@@ -1,13 +1,101 @@
+
+## Contributing
+
+Please read through our [contributing guidelines](https://reponame/blob/master/CONTRIBUTING.md). Included are directions for opening issues, coding standards, and notes on development.
```
Let's modify the URL here.

lautapercuspain commented on README.md:
```diff
@@ -1,13 +1,101 @@
+
+Please read through our [contributing guidelines](https://reponame/blob/master/CONTRIBUTING.md). Included are directions for opening issues, coding standards, and notes on development.
+
+Moreover, all HTML, CSS and TSX files should conform to the [Code Guide](https://github.com/mdo/code-guide), maintained by [@lautapercuspain](https://www.github.com/lautapercuspain).
```
We can link the CONTRIBUTING file here. 👍🏻

lautapercuspain reviewed (commented):
@roznerx  Overall looks good, only few minor changes suggested.

roznerx committed:
Update README.md

Fixed a couple of the reviewed on the file. Now moving onto the CONTRIBUTING.md.

roznerx committed:
Update README.md

Removed the Quick Start section (moving it to the CONTRIBUTING.md file).

roznerx committed:
Update README.md

Added credits for the CONTRIBUTING.md file.

roznerx committed:
Update CONTRIBUTING.md

Initial commit for the CONTRIBUTING.md . Includes the Quick Start section.

roznerx committed:
Update README.md

Updated links to the CONTRIBUTING.md files with proper urls.

roznerx committed:
Update CONTRIBUTING.md

Added ENV data with placeholders; also edited the template to make it more suitable to Code Genius.

roznerx committed:
Update CONTRIBUTING.md

Removed allusions to SemServer.

lautapercuspain committed:
happy coding

lautapercuspain committed:
template deleted

lautapercuspain committed:
Merge branch 'main' into readme-update
